.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
Thanks! Added it below. Any thoughts on testing async dispatches as commented below?